tests/resources/test_catalyst.py

```diff
@@ -173,21 +169,50 @@ def assistant_predictor(assistant_predictor_data):
     return GraphPredictor.build(assistant_predictor_data)
 
 
+@pytest.fixture
+def insights_response_data():
+    return {
+        "response": "Here is the insight based on the literature.",
+        "relevant_documents": True,
+        "document_metadata": [{"source": "doi:10.1234/example", "source_type": "doi"}],
+    }
+
+
+@pytest.fixture
+def insights_response_failed_data():
+    return {
+        {
+            "response": "Error during insight generation.",
+            "relevant_documents": False,
+            "document_metadata": [],
+        }
+    }
```

Comment:
You're not using this fixture anywhere. And it's malformed, as it's a dict within a dict, without a key.